src/main/java/school/lemon/changerequest/java/introduction/hw1/Task2.java

```diff
@@ -3,6 +3,10 @@
 
 public class Task2 {
     public static int getMeters(int centimeters) {
-        return 0;
+        double b = .01;//тут мы вносим дабл 0.01
+        return (int) (centimeters * b);//не забываем поставить вывод через int
```

Comment:
It's quite non-common way to solve this task.
Again, it's great that you've found interesting solution, but try to use more common one, cause you should also think about your team members and this solution is harder to read, than simple `centimeters/100`